scripts/update-release-metadata.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+import json
+import sys
+import boto3
+import re
+
+
+def fetch_metadata():
+  '''
+  Pull metadata from s3 as byte stream.
+  '''
+  client = boto3.client('s3')
+  try:
+    resp = client.get_object(
+        Bucket='release-metadata',
+        Key='release-versions'
+    )
+  except boto3.S3.Client.exceptions.NoSuchKey:
+    sys.stderr.write('No release-metadata key found.  Check permissions.\n')
+    return 1
+
+  return resp['Body'].read()
+
+
+def write_metadata(data):
+  '''
+  Write metadata file from byte stream.
+  '''
+  client = boto3.client('s3')
+  resp = client.put_object(
+      Body=data,
+      Bucket='release-metadata',
+      Key='release-versions'
+  )
+
+  if resp.ResponseMetadata.HTTPStatusCode != 200:
+    sys.stderr.write('Unable to upload file.  Dumping response metadata.\n')
+    print(resp, file=sys.stderr)
```

Comment:
Just a sanity check, does this metadata risk having any auth info in it? If so, we shouldn't be dumping it in the output of actions (especially in a public repo).

Reply:
I don't think so, the only identifiable information is a hash of the uploaded object.

Reply:
For reference,
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3.Client.put_object